content/graphql/quick-start.md

```diff
@@ -58,6 +58,8 @@ export class AppModule {}
 
 As mentioned, these options will be forwarded to the `ApolloServer` constructor.
 
+> warning **Warning** If you use `apollo-server-fastify` package, enable in the `GraphQLModule` configuration the `disableHealthCheck` setting to `true`.
```

Comment:
This happens only if you have multiple GraphQL endpoints in a single app.

Reply:
Ok 👍, I made the fix